esp/platform/espsecurecontext.hpp

```diff
@@ -105,6 +105,8 @@ inline bool IEspSecureContext::getProp(int type, const char* name, TValue& value
     return found;
 }
 
+//ESP may use the IEspSecureContextEx to populate some data. A security
+//manager may read (not change) the data using the IEspSecureContext.
```

Comment:
This is close, but not quite accurate since there is a set function. Perhaps something like:

*Where IEspSecureContext defines a security manager's interface to request data, this extension defines the ESP's interface to setup the data for the manager."

This is just an idea. You don't have to use it.